ptn/boozebot/constants.py

```diff
@@ -243,14 +243,14 @@ def get_db_path():
     return CARRIERS_DB_PATH
 
 
-def server_admin_role_id():
+def server_council_role_ids():
     """
-    Wrapper that returns the admin role ID
+    Wrapper that returns the council role IDs
 
-    :returns: Admin role id
-    :rtype: int
+    :returns: Council role ids
+    :rtype: Tuple[int, ...]
```

Comment:
nitpick(non-blocking): since python3.9 you can use `tuple[int, ...]`, `list`, `dict` directly, without typing module (you are not even importing it)

it would be even better to annotate the function itself, rather than the docstring (even doc-gen tools like sphinx support type hints for autodoc)
```
def server_council_role_ids() -> tuple[int, ...]:
```